public/app/features/admin/Users/OrgUsersTable.tsx

```diff
@@ -107,7 +108,9 @@ export const OrgUsersTable = ({
       {
         id: 'lastSeenAtAge',
         header: 'Last active',
-        cell: ({ cell: { value } }: Cell<'lastSeenAtAge'>) => value,
+        cell: ({ cell: { value } }: Cell<'lastSeenAtAge'>) => {
+          return <>{value && <>{value === '10 years' ? <Text color={'disabled'}>Never</Text> : value}</>}</>;
```

Comment:
```suggestion
          return <>{value && value === '10 years' ? <Text color={'disabled'}>Never</Text> : value}</>;
```

Reply:
changed in the `OrgUsersTable.tsx` and the `UsersTable.tsx`

Reply:
@eleijonmarck Let me correct myself, the change that I recommended changes the result if `value === undefined`, so your first solution was right, could you please revert this change and the other one? Sorry 😇

Reply:
@mgyongyosi i tested out the changes locally 🤔 i think. let me first try locally again